addon/adapters/user.js

```diff
@@ -1,4 +1,20 @@
 import OsfAdapter from './osf-adapter';
+import Ember from 'ember';
 
 export default OsfAdapter.extend({
+    findHasMany(store, snapshot, url, relationship) {
```

Comment:
Per version differences: long-term, I'd encourage us to move `ember-osf` to Ember 2.8 (which was released earlier this month, and is currently a candidate LTS build with a more stable support commitment).

However, I also recognize that upgrading ember versions is not a trivial task; it's a separate ticket in itself. We should probably create such a ticket (and note this as an area to change). But so long as we're pinned to an old ember version this approach is fine for now.